cmd/neofs-node/config.go

```diff
@@ -989,15 +966,6 @@ func (c *cfg) configWatcher(ctx context.Context) {
 				continue
 			}
 
-			for _, component := range components {
-				err = component.cfg.Reload()
```

Comment:
I'm not sure what components are affected by this change. Are we gonna be able to reconfigure log level the way it was possible before?

Reply:
yep, `configWatcher` still reconfigures logger but in a different way (decoding `zap.AtomicLevel`)